### DISCUSSION TITLE
What is the transaction type of extension prisma?

### DISCUSSION THREAD

**Discussion by greatbooms** [prisma/prisma#21819] [Q&A]
### Question

![스크린샷 2023-11-07 오전 11 50 44](https://github.com/prisma/prisma/assets/5744408/3089b98e-a0f6-4d97-bec0-f96f986bf2f4)
![스크린샷 2023-11-07 오전 11 51 37](https://github.com/prisma/prisma/assets/5744408/5e505195-f8f1-429d-993f-f570a3f271fd)

What is the transaction type for scalable prisma?

The type is not known, so type inference is not done correctly in the code below.

I would like to know the solution.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @greatbooms 👋 

Thank you for raising this question.

We do have an open [issue](https://github.com/prisma/prisma/issues/20738) to support Extended Prisma Client on `TransactionClient` type. There is also a workaround to infer the type as shown [here](https://github.com/prisma/prisma/issues/20738#issuecomment-1807917019).

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
